src/components/SearchToolbar/index.js

```diff
@@ -0,0 +1,21 @@
+import "./SearchToolbar.scss";
+
+export const SearchToolbar = ({ city, country, setData }) => {
+	const handleCitySearch = (event) => {
+		if (event.key === "Enter") {
+			setData(prev => ({
```

Comment:
it's not a big deal, but u can set new state as setState({...state, city: event.target.value}). Is your changing of the city fetching new temperature?? if no u should add that functionality

Reply:
Yeah, changing city fetching new temperature